### DISCUSSION TITLE
Sound Sensor And LED Bar | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#130] [Q&A]
# Sound Sensor And LED Bar | Seeed Studio Wiki

Sound Sensor And LED Bar

https://wiki.seeedstudio.com/Sound_Sensor_And_LED_Bar/

**Comment by ghost** [comment]
where is the code

**Reply by MatthewJeffson** [reply]
That, is an interesting question... I will check with my colleague. If they still don't have it, I will put this on the task list, looking for the contribution. Thank you for pointing it!
